frontend/src/shared/components/Toast/store/toastActions.ts

```diff
@@ -0,0 +1,103 @@
+import { DEFAULT_DURATION_MS, DEFAULT_TYPE } from "../constants/toast.constants";
+import type { ToastData, ToastOptions, ToastsState } from "../types/toast.type";
+import { toastStore } from "./toastStore";
+
+const getDistributedToasts = (state: ToastsState) => {
+  const { toasts, defaultPosition, limit } = state;
+  const queue: ToastData[] = [];
+  const activeToasts: ToastData[] = [];
+  const count: Record<string, number> = {};
```

Comment:
```suggestion
  const count: Record<ToastPosition, number> = {};
```
`position` 키 값에 대한 타입을 좁혀도 좋을 것 같네요🙃